app/Console/Kernel.php

```diff
@@ -23,11 +23,12 @@ class Kernel extends ConsoleKernel
 
     /**
      * Define the application's command schedule.
-     *
-     * @return void
      */
-    protected function schedule(Schedule $schedule)
+    protected function schedule(Schedule $schedule): void
     {
+        $schedule->command('cache:prune-stale-tags')
```

Comment:
Why the need to change which scheduled command is run?

Reply:
Not a change here, but an addition of the `cache:prune-stale-tags` command based on new Redis functionality. More here: https://laravel.com/docs/10.x/upgrade#redis-cache-tags